.github/workflows/build-wheels.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       - name: Build distro
         env:
           ZROK_VERSION: ${{ github.event.release.tag_name }}
-          ZROK_PY_NAME: ${{ vars.ZROK_PY_NAME || null }}
```

Comment:
This is still useful, at least for Ken, for building the project in a fork. It allows the forkee to override the PyPi package name with a GitHub Actions environment variable. That way, committing a code change is unnecessary in order to verify that the PyPi build and release machinery is working before sending a pull request to the upstream zrok maintainers.

Reply:
Re-added!